### ISSUE TITLE
Consider a cloud CI service

### ISSUE THREAD

**Issue by robertoostenveld** [fieldtrip/playground#14]
This follows up on https://github.com/fieldtrip/fieldtrip/issues/1758 which was raised by @acampbel

**Comment by robertoostenveld** [comment]
I have set up travis, erased the old `.travis.yml`, created a new one that only contains

```
language: matlab
```
 
Nothing happens, neither when I do a PR, nor when I merge with master. Prior to getting to this level with the minimalistic travis.yaml, we already had some workflows. Earlier today I added one of them, and that worked. See 
https://github.com/fieldtrip/playground/commit/83e3818c9d0624bee088637e88664b7047c50acd#diff-a1e8f40da635dd08be3102a768b38f66c5cb018e5aa60289b38035af099a9729

**Comment by robertoostenveld** [comment]
@acampbel could you clarify how the travis job gets started? If I go to https://travis-ci.com/github/fieldtrip/playground I can see the old jobs (~10 months ago), but no new jobs.

**Comment by robertoostenveld** [comment]
I added the `.github/workflows/matlab.yml` file, and am able to run it from the Actions menu, where it says "This workflow has a workflow_dispatch event trigger". Subsequently it runs, and shows the expected results.

I don't have the impression that this runs at Travis, as on the travis dashboard I don't see any reference to it. Is this then running somewhere in the Azure cloud or so?

I will remove Travis permissions from the repository, and try again...

**Comment by robertoostenveld** [comment]
... yes, after removing Travis altogether, I can still trigger the action. I now also see in the logs that the action makes use of `http://azure.archive.ubuntu.com/` to prepare the system for MATLAB. So it indeed seems to run in Azure. 

The simplified syntax that according to https://blogs.mathworks.com/developer/2020/12/15/cloud-ci-services should work for Travis does apparently not apply. I still don't understand why I cannot get Travis back to work.

**Comment by robertoostenveld** [comment]
Ah, after manually "trigger build" on travis, I see "Could not authorize build request for fieldtrip/playground", which brought me to https://stackoverflow.com/questions/41034694/travis-could-not-authorize-build-request/41078031

Following instructions there, I set up the free plan. Initially only for my personal account (still leading to the same error for fieldtrip/playground), then also for the fieldtrip organization. Subsequently the "trigger build" with a custom "language: matlab" yaml is doing something, and gives me 

```
The MATLAB language is maintained by MathWorks.
If you have any questions or suggestions, please contact MathWorks at continuous-integration@mathworks.com.
matlab_install
Installing MATLAB
...
$ matlab -batch "assertSuccess(runtests('IncludeSubfolders',true));"

```

That is what I was expecting! 
So the problem was that for my (already existing) travis account I did not have a (Free) plan selected.
